hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -194,6 +193,20 @@ class HassioAddonInfo extends LitElement {
                 : html` ${this.addon.version_latest} `}
             </div>
           </div>
+          <div class="description light-color">
+            ${this.addon.version
+              ? html`
+                  Current version: ${this.addon.version}
+                  <div class="changelog" @click=${this._openChangelog}>
+                    <span>(</span><span class="changelog-link">changelog</span
+                    ><span>)</span>
```

Comment:
Why the spans around the `()`?

Reply:
To stop autoformatting on adding weird spaces.
Do you have another solution for that?